packages/amazonq/.changes/next-release/Bug Fix-7bd6029d-04bc-4135-890a-1a8e4f70c18c.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Disable LSP in AL2"
```

should we mention "Amazon Q" specifically, to avoid any user confusion about other LSPs?
```suggestion
	"description": "Disable Amazon Q LSP in AL2"
```